src/control/server/storage_scm.go

```diff
@@ -80,30 +112,127 @@ func (s *scmStorage) Discover() error {
 		s.modules = pbMms
 		return nil
 	}
-	return fmt.Errorf("scm storage not initialized")
+	return errors.Errorf("scm storage not initialized")
 }
 
-// todo: implement remaining methods for scmStorage
-// func (s *scmStorage) Update(params interface{}) interface{} {return nil}
-// func (s *scmStorage) BurnIn(params interface{}) (fioPath string, cmds []string, env string, err error) {
-// return
-// }
+// clearMount unmounts then removes mount point.
+//
+// NOTE: requires elevated privileges
+func (s *scmStorage) clearMount(mntPoint string) (err error) {
+	if err = s.config.ext.unmount(mntPoint); err != nil {
+		return
+	}
 
-// Setup placeholder implementation for scmStorage
-func (s *scmStorage) Setup() error {
-	s.initialized = true
-	return nil
+	if err = s.config.ext.remove(mntPoint); err != nil {
+		return
+	}
+
+	return
 }
 
-// Teardown placeholder implementation for scmStorage
-func (s *scmStorage) Teardown() error {
-	s.initialized = false
+// reFormat wipes fs signatures and formats dev with ext4.
+//
+// NOTE: Requires elevated privileges and is a destructive operation, prompt
```

Comment:
Where is the prompt for confirmation handled?

Reply:
it is handled on the client side in the management tool (daos_shell), it calls this method over gRPC using the client API. This comment is a note to say that the confirmation should be handled by the calling code.